kclvm/error/src/lib.rs

```diff
@@ -39,12 +42,17 @@ pub struct Handler {
     /// This is not necessarily the count that's reported to the user once
     /// compilation ends.
     emitter: Box<dyn Emitter>,
+    /// A switch to hide panic messages.
+    /// The default value is false, when an error occurs, the error message will be filled into 'PanicInfo' and panic out.
+    /// If the 'hide_panic_info' is true, when an error occurs, the error message will not be raised by panic.
+    hide_panic_info: bool,
```

Comment:
What we need to do is to use `PanicInfo` as the KCL parser and exec_program error returns instead of hiding it.